### PR TITLE
Change Docker Buildx and QEMU setup actions to official ones

### PR DESCRIPTION
**Pull request by mtyazici**
There are official Github Actions for setting up QEMU and Docker Buildx. This PR removes the old action and adds the new ones into the workflows.

### CODE REVIEW

File: .github/workflows/latest_image_push.yml [hunk 1]
```diff
@@ -12,12 +12,13 @@ jobs:
       - name: Checkout Code
         uses: actions/checkout@v2
 
-      - name: Set up Docker Buildx
-        id: buildx
-        uses: crazy-max/ghaction-docker-buildx@v3.3.1
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1.0.1
```
Comment (hasancelik) [inline .github/workflows/latest_image_push.yml:16]: What is QEMU version? Is it configurable separately?

Comment (mtyazici) [reply .github/workflows/latest_image_push.yml:16]: Unfortunately, the only configurable fields are `image` and `platforms`. Default image is : `tonistiigi/binfmt:latest`.

Comment (hasancelik) [reply .github/workflows/latest_image_push.yml:16]: I see. It seems latest image utilizes QEMU 5.0.0 currently:

Screen Shot 2021-01-07 at 14 35 14

**Review by hasancelik — APPROVED** [review]
👍

**Pull request merged**